devices/iluminize/led_controller_multi_5.rgbw.json

```diff
@@ -0,0 +1,186 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "iluminize",
+  "modelid": "HK-ZD-RGB-A",
+  "product": "LED Controller Multi 5 (RGBW mode)",
```

Comment:
I don’t know the device, but typically RGBW controllers expose two endpoints: a _Color Light_ for RGB and a _Dimmable Light_ for W.  I doubt it supports `ct` in this mode.

Reply:
These endpoints are active and show changing numbers. If I remember correctly they are not writable. The controller seems to do some internal mix of colors and is optimized for neutral white LEDs (4000 kelvin). All Philips RGBW lamps I had up to now do the same and don't provide a dedicated white dimmer.

Reply:
Correct, the endpoints and clusters are reported by the device and cannot be written.

What Philips lights are you referring to?  Afaik, Hue don’t have RGBW lights.

There are other LED controllers that use multiple endpoints, like GLEDOPTO.


But anyways, if the device supports `ct` in  RGBW mode, we should expose it as a _Color Dimmable Light_.

Reply:
> What Philips lights are you referring to? Afaik, Hue don’t have RGBW lights.

In some Philips HUE I disassembled also have been RGBW LEDs (maybe it was Xamento), but the controller does not provide a dedicated cluster to control the white ones. So I assume it also does some mix which matches the colorspace of the LEDs. (Annoying here: in white colors it's freaking bright and while using saturated colors, it is very dark.)